Userland/Libraries/LibVideo/VP9/Decoder.h

```diff
@@ -33,16 +46,101 @@ class Decoder {
     u8 adapt_prob(u8 prob, u8 counts[2]);
 
     /* (8.5) Prediction Processes */
-    DecoderErrorOr<void> predict_intra(size_t plane, u32 x, u32 y, bool have_left, bool have_above, bool not_on_right, TXSize tx_size, u32 block_index);
-    DecoderErrorOr<void> predict_inter(size_t plane, u32 x, u32 y, u32 w, u32 h, u32 block_index);
+    // (8.5.1) Intra prediction process
+    DecoderErrorOr<void> predict_intra(u8 plane, u32 x, u32 y, bool have_left, bool have_above, bool not_on_right, TXSize tx_size, u32 block_index);
+
+    // (8.5.1) Inter prediction process
+    DecoderErrorOr<void> predict_inter(u8 plane, u32 x, u32 y, u32 w, u32 h, u32 block_index);
 
     /* (8.6) Reconstruction and Dequantization */
-    DecoderErrorOr<void> reconstruct(size_t plane, u32 transform_block_x, u32 transform_block_y, TXSize transform_block_size);
+
+    u16 dc_q(u8 b);
+    u16 ac_q(u8 b);
+    /* Returns the quantizer index for the current block */
+    u8 get_qindex();
+    /* Returns the quantizer value for the dc coefficient for a particular plane */
+    u16 get_dc_quant(u8 plane);
+    /* Returns the quantizer value for the ac coefficient for a particular plane */
+    u16 get_ac_quant(u8 plane);
+
+    /* (8.6.2) Reconstruct process */
+    DecoderErrorOr<void> reconstruct(u8 plane, u32 transform_block_x, u32 transform_block_y, TXSize transform_block_size);
+
+    /* (8.7) Inverse transform process */
+    DecoderErrorOr<void> inverse_transform_2d(Vector<Intermediate>& dequantized, u8 log2_of_block_size);
+
+    /* (8.7.1) 1D Transforms */
+    /* (8.7.1.1) Butterfly functions */
+
+    inline i32 cos64(u8 angle);
+    inline i32 sin64(u8 angle);
+    /* The function B( a, b, angle, 0 ) performs a butterfly rotation. */
+    inline void butterfly_rotation_in_place(Vector<Intermediate>& data, size_t index_a, size_t index_b, u8 angle, bool flip);
+    /* The function H( a, b, 0 ) performs a Hadamard rotation. */
```

Comment:
This file has a mix of `//` and `/* */` style comments, which seems to be somewhat pre-existing? 

However in commit `LibVideo: Implement VP9 intra-predicted frame decoding` you converted some to `//` and added a ton of `/* */` comments. Could we stick to new comments (outside of the file copyright header, for uh... reasons) being `//` and come back later (like a later PR) to fully clean up the c style comments that were pre-existing (when you don't touch that function anyway)?

Reply:
Fixed those, turns out I had a few more /* laying around as well that I've fixed.